polysemy-RandomFu/src/Polysemy/RandomFu.hs

```diff
@@ -89,11 +91,11 @@ runRandomIO = interpret $ \case
 ------------------------------------------------------------------------------
 -- | Run in 'IO', using the given 'R.PureMT' source, stored in an 'IORef'
 runRandomIOPureMT
-  :: MonadIO (Sem r)
+  :: Member (Lift IO) r
   => R.PureMT
   -> Sem (RandomFu ': r) a
   -> Sem r a
 runRandomIOPureMT source re =
-  liftIO (newIORef source) >>= flip runRandomSource re
+  sendM (newIORef source) >>= flip runRandomSource re
```

Comment:
See above.  I think the ```MonadIO``` instance for ```Member (Lift IO) r => Sem r``` is just ```liftIO = sendM``` so these versions are equivalent.  Am I missing something?

Reply:
Yeah this change isn't necessarily needed, but I thought it made more sense this way with the changed constraint above, since the `Lift` effect is usually used with `sendM`, while `liftIO` is part of `MonadIO`.

Reply:
This makes sense.  Thanks!